README.md

```diff
@@ -63,4 +63,16 @@ If you have any issues or just wanna chat you can reach us on our [Discord Serve
 ### Testing the SDK
 Status: [![Test SDK with Unity](https://github.com/LootLocker/unity-sdk/actions/workflows/run-tests-and-package.yml/badge.svg?branch=main)](https://github.com/LootLocker/unity-sdk/actions/workflows/run-tests-and-package.yml?query=branch%3Amain)
 
-There is a Test Suite for the SDK, but it is in an [external repo](https://github.com/LootLocker/unity-sdk-tests) to keep the size of this one down. It is run automatically on any pull requests towards or updates to main. You can also run it locally, just follow the steps in the test repo.
+To set up the tests, you must add our package to the manifest.json `testables` underneath `dependencies`. Your manifest.json should look something like this:
+
+```json
+{
+  "dependencies": {
+    "com.lootlocker.lootlockersdk": "https://github.com/lootlocker/unity-sdk.git",
+    many more dependencies...
+  },
+  "testables": ["com.lootlocker.lootlockersdk"]
+}
+```
+
+Once you have done this, you can go into Unity and go to `Window` -> `General` -> `Test Runner`. This will open a new window which should already include all the tests available, Then all you have to do is press `Run All` and the tests will begin.
```

Comment:
Maybe we should write something about how to configure which user it uses, that you will be rate limited if you're not running your own environment, etc. etc.

Reply:
Fixed